scissors/Makefile

```diff
@@ -0,0 +1,9 @@
+CC = gcc
+CFLAGS = -g
+NAME = game
+$(NAME):weapon_selection.c game_logic.c main.c
+
+	$(CC) $(CFLAGS) -o $@ $^
+clean:
```

Comment:
The same as in .c files: use empty lines to visually separate blocks (variable block, between rules, etc).